src/en/infinityscans/src/eu/kanade/tachiyomi/extension/en/infinityscans/InfinityScans.kt

```diff
@@ -49,59 +47,114 @@ class InfinityScans : HttpSource() {
         add("Sec-Fetch-Dest", "empty")
         add("Sec-Fetch-Mode", "cors")
         add("Sec-Fetch-Site", "same-origin")
-        add("X-Requested-With", "XMLHttpRequest")
+        add("X-requested-with", "XMLHttpRequest")
     }.build()
 
     private val json: Json by injectLazy()
 
     // Popular
+    override fun fetchPopularManga(page: Int): Observable<MangasPage> {
```

Comment:
Use the request and parse methods instead

Reply:
Ack,
fetchPopularManga
fetchLatestUpdates
reverted to use request and parse methods

fetchSearchManga is impossible to revert as in parse method we don't know about query or filters.